pyproject.toml

```diff
@@ -1,8 +1,11 @@
 [project]
 name = "qp-prob"
+requires-python = ">=3.9"
 license = {file = "LICENSE"}
 readme = "README.md"
-
+authors = [
+    { name = "LSST Dark Energy Science Collaboration (DESC)", email = "later@later.com" }
```

Comment:
This is where `later@later.com` ended up. It's not considered best practice, but since `author_email` is only used in one location, you can update it here and in the .copier-answeres.yml file and avoid having to run `copier update` again.